packages/wrangler/src/pages/dev.tsx

```diff
@@ -24,6 +24,10 @@ const DURABLE_OBJECTS_BINDING_REGEXP = new RegExp(
 	/^(?<binding>[^=]+)=(?<className>[^@\s]+)(@(?<scriptName>.*)$)?$/
 );
 
+const SERVICE_BINDING_REGEXP = new RegExp(
+	/^(?<binding>[^=]+)=(?<service>[^@\s]+)(@(?<environment>.*)$)?$/
```

Comment:
I'd double-check the environments bit here. Might be totally fine, but I wouldn't be surprised if there's weirdness where I need to do something like:

```
--service MyBinding=my-worker--local
```

instead of

```
--service MyBinding=my-worker@local
```

If there is anything funky, I'd suggest dropping support for passing in an environment right now, just until we can think over how we want them to work (esp. given environments v2).

`wrangler pages dev --do` Durable Objects don't support environments today because of this. Let's just make sure the solution works across the board.

Reply:
Yep you're right, the environment gets appended onto the end
> "service-test-dev"
`wrangler dev --local --persist --env dev`

Should be easy to support, i'll do a quick try and if it fails I'll just leave it for now.